viite-UI/src/model/SelectedLinkProperty.js

```diff
@@ -50,6 +50,10 @@
       current = data;
     };
 
+    var idTestForOpen = function (id) {
```

Comment:
This name could be something like "canOpenById"

Reply:
I think that what you want to check here is if you have a valid id on the road address could be:
`return !_.isUndefined(id) && id > 0;`
And the name can be `hasValidId`